CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - [appengine_api] Add metadata to device
+- [trigger_engine] Allow configuring preferred http method (such as put or get) (see gh #128).
+- [trigger_egnine] Add optional support to custom http headers, such as
+  "Authorization: Bearer ..." (see gh #129).
```

Comment:
Use markdown links (e.g. `[#129](https://github.com/astarte-platform/astarte/issues/129)`) to link in the CHANGELOG, because GitHub magic links don't work here

Reply:
And use backticks for `Authorization: Bearer ...`